### DISCUSSION TITLE
/zh-cn/blog/posts/night_talk_003

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#144] [Announcements]
# 音乐与时间与蓝色星球#03 拍星星 - Reverier's Blog

前言 有人想不开要学双拼输入法.jpg 这篇博客打算全部用双拼写，看看我能坚持多久，要是中途放弃了，我会标出来在哪里放弃的。 摄影 接上文，最后还是

https://blog.woooo.tech/posts/night_talk_003/

**Comment by LittFlower** [comment]
好看耶！
